.travis.yml

```diff
@@ -1,11 +1,14 @@
 language: python
 dist: xenial
-matrix:
+jobs:
   include:
-    - python: 2.7
-    - python: 3.5
+    - name: Minimum NEP 029 versions
```

Comment:
What is this?

Reply:
Forgot to include the link.  NEP 029 requires support for minimum previous versions for both python and numpy on a rolling timeline.  This is part of the upcoming PyHC requirements for code.  
https://numpy.org/neps/nep-0029-deprecation_policy.html

So between now and June, python 3.6 is the minimum version.  Between now and July, numpy 1.15 is the minimum.